docs/getting-started.md

```diff
@@ -50,20 +61,26 @@ $ cd docs/getting-started
 First of all, you need to define how the data will be organized (a.k.a database schema) in the application with Scalar DB database schema.
 Here is a database schema for the sample application. For the supported data types, please see [this doc](schema.md) for more details.
 
-```sql
-REPLICATION FACTOR 1;
-
-CREATE NAMESPACE emoney;
-
-CREATE TABLE emoney.account (
-    id TEXT PARTITIONKEY,
-    balance INT,
-);
+```json
+{
+  "emoney.account": {
+    "transaction": false,
+    "partition-key": [
+      "id"
+    ],
+    "clustering-key": [],
+    "columns": {
+      "id": "TEXT",
+      "balance": "INT"
+    },
+    "compaction-strategy": "STCS"
+  }
+}
 ```
 
 To load the schema file, please run the following command.
 ```
-$ $SCALARDB_HOME/tools/schema/loader emoney-storage.sdbql
+$ java -jar $SCALARDB_HOME/tools/scalar-schema/scalar-schema.jar --cassandra -h localhost -u <CASSNDRA_USER> -p <CASSANDRA_PASSWORD> -f emoney-storage.json -R 1
```

Comment:
Please add the jar to the releases and use it. (so we need a download step)
I think each release should have the jar built with the release.

Reply:
Thank you for your suggestion.
Document updated with the above suggestion.

scalar-schema-2.2.1.jar available in [scalardb release v2.2.1](https://github.com/scalar-labs/scalardb/releases/tag/v2.2.1)

`scalar-schema.jar` updated with scalardb release version number. `v` is not using with jar file because the source (zip) file does not contain `v`,  it downloads as `scalardb-2.2.1.zip`